### ISSUE TITLE
Designing Audio Equalization Filters by Deep Neural Networks

### ISSUE THREAD

**Issue by thereisatablehere** [thereisatablehere/CapstoneResources#1]
### Summary
Audio equalization is an active research topic aiming at improving the audio quality of a loudspeaker system by correcting the overall frequency response using linear filters. The estimation of their coefficients is not an easy task, especially in binaural and multipoint scenarios, due to the contribution of multiple impulse responses to each listening point. This paper presents a deep learning approach for tuning filter coefficients employing three different neural networks architectures—the Multilayer Perceptron, the Convolutional Neural Network, and the Convolutional Autoencoder. Suitable loss functions are proposed for each architecture, and are formulated in terms of spectral Euclidean distance. The experiments were conducted in the automotive scenario, considering several loudspeakers and microphones. The obtained results show that deep learning techniques give superior performance compared to baseline methods, achieving almost flat magnitude frequency response.

### Key Points 
- FIR filter 
- Deep Neural Networks
- Spectral Analysis

### Citation
P. G. Pepe, L. Gabrielli, S. Squartini, and L. Cattani, "Designing Audio Equalization Filters by Deep Neural Networks," _Applied Sciences_, vol. 10, no. 7, p. 2483, 2020. [Online]. Available: https://doi.org/10.3390/app10072483.

### Repo link
N/ A
